src/app/components/search/search.component.ts

```diff
@@ -59,7 +60,7 @@ export class SearchComponent implements OnInit, AfterViewChecked, OnDestroy {
   courthouses$ = this.caseService.getCourthouses();
   subs: Subscription[] = [];
 
-  constructor(private caseService: CaseService) {}
+  constructor(private caseService: CaseService, private userService: UserService) {}
```

Comment:
I don't think this is referenced anywhere.

Reply:
I think this service needs to be injected into the hearing.component.ts instead

Reply:
Removed